Makefile

```diff
@@ -32,13 +33,13 @@ generate-proto: ## regenerate protos
 	@echo " > protobuf compilation finished"
 
 unit-test:
-	go list ./... | grep -v -e third_party -e api/proto | xargs go test -count 5 -cover -race -timeout 1m -tags=unit_test
+	go test -count 1 -cover -race -timeout 1m -tags=unit_test ./...
```

Comment:
Count `5` was used to try to execute a unit test more than once to catch code with undefined behaviours. I see we have reduced it to `1` here. Any reason we don't want to run these test as before?